src/Pages/Login/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { FormEvent, useContext, useEffect } from "react";
+import Header from "../../Components/Header";
+import UserContext from "../../Contexts/UserContext";
+import SubmitInput from "../../Utilities/SubmitInput";
+import BASE_URL from "../../settings";
+import { useNavigate } from "react-router";
+
+function Login() {
+  const { userId, changeUserId } = useContext(UserContext);
+  const navigate = useNavigate();
+
+  async function userSignIn(formData: FormData) {
+    let data = {
+      email: formData.get("email"),
+    };
+
+    try {
+      const response = await fetch(`${BASE_URL}/users`, {
+        method: "POST",
+        body: JSON.stringify(data),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+      const responseData = await response.json();
+      if (response.ok) {
+        changeUserId(responseData.data.userId);
+      } else {
+        console.log(responseData.message);
+      }
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  function handleSubmit(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    const formData = new FormData(e.currentTarget);
+    userSignIn(formData);
+  }
+
+  useEffect(() => {
+    if (userId) {
+      navigate(`/recipes/${userId}`);
+    }
```

Comment:
Negative numbers will also type-juggle to `true` but I don't think this should be correct since it will break things if you ask the database for a negative id. There's a simple check that would be more robust.

Reply:
Done